hello.py

```diff
@@ -1 +1,6 @@
-print("Hello, world!")
+def hello():
+    print("Hello, world!")
+
+if __name__ == '__main__':
```

Comment:
I'd like for this project to use PEP 8 formatting. So four-space indents!

```suggestion
if __name__ == "__main__":
```